workshops/models.py

```diff
@@ -209,7 +209,7 @@ class Person(AbstractBaseUser, PermissionsMixin):
     )
 
     # These attributes should always contain field names of Person
-    PERSON_UPLOAD_FIELDS = ('personal', 'middle', 'family', 'email')
```

Comment:
Because, as stated in commit message for ee2b234, it seems like a common mistake for admins to, for example, insert last name into middle name field.